components/atoms/Select/custom-select.tsx

```diff
@@ -10,72 +14,59 @@ interface SelectProps {
   className?: string;
 }
 
-type SelectedState = {
-  name: string;
-  value: any;
-};
-const Select = ({
+const CustomSelect = ({
   placeholder = "Select an option",
   options,
-  error, // don't know if this is necessary
   label,
   className,
   onChange
 }: SelectProps): JSX.Element => {
-  const [isOpen, setIsOpen] = useState<boolean>(false);
-  const [selected, setSelected] = useState<SelectedState>({ name: "", value: "" });
-  const handleToggle = () => {
-    setIsOpen(!isOpen);
-  };
-  const handleSelected = (option: { name: string; value: any }) => {
-    const { name, value } = option;
-    setSelected((prev) => ({ ...prev, name, value }));
-    onChange?.(value);
-    setIsOpen(false);
-  };
 
   return (
-    <div
-      className={`${
-        className
-          ? className
-          : "relative bg-white cursor-pointer  focus:border-light-orange-9 focus:ring focus:ring-light-orange-5 items-center overflow-x-hidden  rounded-lg   text-base text-light-slate-10"
-      }`}
-    >
-      <div
-        onClick={() => handleToggle()}
-        className="flex px-4 py-1.5 bg-white border border-light-slate-6 rounded-lg focus-within:border-light-orange-9 focus-within:ring focus-within:ring-light-orange-5 items-center"
-      >
-        {label && <span className="text-sm inline-flex text-light-slate-9 mr-2">{label}:</span>}
-        <input
-          value={selected.name === "" ? placeholder : selected.name}
-          readOnly
-          type="text"
-          className="overflow-scroll text-sm cursor-pointer text-light-slate-12 focus:outline-none bg-transparent"
-        />
-        <div className="w-6 h-4 relative overflow-hidden">
-          {" "}
-          <RiArrowUpSLine className="absolute bottom-1" /> <RiArrowDownSLine className="absolute top-1  " />
-        </div>
+    <Select.Root>
+      <div className={clsx("relative bg-white cursor-pointer focus:border-light-orange-9 focus:ring focus:ring-light-orange-9 items-center overflow-x-hidden rounded-lg text-base text-light-slate-10 min-w-min", className ? className : "")}>
```

Comment:
nit: not a blocker but a suggestion:
use the `||` operator instead of the ternary `?`
```suggestion
      <div className={clsx("relative bg-white cursor-pointer focus:border-light-orange-9 focus:ring focus:ring-light-orange-9 items-center overflow-x-hidden rounded-lg text-base text-light-slate-10 min-w-min", className || "")}>
```